ceed-vec.c

```diff
@@ -61,6 +61,22 @@ int CeedVectorSetArray(CeedVector x, CeedMemType mtype, CeedCopyMode cmode,
   return 0;
 }
 
+/// Set the array used by a vector, freeing any previously allocated array if applicable.
```

Comment:
Bad copy-pasta. We'll eventually need this to be implementable by backends (to avoid moving data to/from host). You could add a method pointer and have this implementation use the default when the method pointer is NULL.

Reply:
Updated